app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -18,23 +19,34 @@ import '../request_handling/api_request_handler.dart';
 import '../request_handling/authentication.dart';
 import '../request_handling/body.dart';
 import '../request_handling/exceptions.dart';
+import '../service/datastore.dart';
 
 @immutable
 class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
   const UpdateTaskStatus(
     Config config,
-    AuthenticationProvider authenticationProvider,
-  ) : super(config: config, authenticationProvider: authenticationProvider);
+    AuthenticationProvider authenticationProvider,{
+    @visibleForTesting
+        this.datastoreProvider = DatastoreService.defaultProvider,
+    }) : super(config: config, authenticationProvider: authenticationProvider);
+
+  final DatastoreServiceProvider datastoreProvider;
 
   static const String taskKeyParam = 'TaskKey';
   static const String newStatusParam = 'NewStatus';
   static const String resultsParam = 'ResultData';
   static const String scoreKeysParam = 'BenchmarkScoreKeys';
+  
+  /// const variables for [BigQuery] operations
```

Comment:
Nit: Although this isn't flutter/flutter repo, but it's nice to use the correct grammar. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar